app/bundles/EmailBundle/Config/config.php

```diff
@@ -1,6 +1,5 @@
 <?php
-
-/*
+/**
```

Comment:
We had to replace `/**` with `/*` for file comments because PHP CS Fixer was applying method annotation styles on it on Ubuntu devices and not on others. So whenever I (with Ubuntu) edited a file after someone who edit it with Mac, it added/remove a line for no reason. `/*` is also the correct way for file comments for Symfony code style.

Double check if you use the pre-commit hook (https://github.com/mautic/mautic/blob/staging/.github/CONTRIBUTING.md#coding-standards) to save you some time  with code standards issues.